ports/stm/boards/nucleo_f446re/board.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 flom84 for Adafruit Industries
```

Comment:
Unless you were being paid by Adafruit industries for this work, don't write "for Adafruit Industries" here. The same comment applies through the rest of this PR.

Reply:
okay.  would do the change. thanks